### ISSUE TITLE
Is there a method to serialize the CellTree object to disk?

### ISSUE THREAD

**Issue by barpaum** [NOAA-ORR-ERD/cell_tree2d#10]
I've tried dill and pickle for serialization but failed. Is there any method to save a CellTree to disk so it would be possible to load it for a second time use.

**Comment by jay-hennen** [comment]
No, there isn't a mechanism for that right now. For our primary use case, building it anew every time is acceptable.

However, do feel free to submit a PR for it! It would definitely be a nice feature to have.
